src/styles/block/watch.scss

```diff
@@ -0,0 +1,47 @@
+.watch {
+  position: relative;
+  height: 500px;
+  width: 500px;
+
+  border-radius: 50%;
+  border: 10px dashed #000;
+
+  &__minute,
+  &__center,
+  &__second {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+  }
+
+  &__minute {
+    width: 20px;
+    height: 125px;
+
+    background-color: blue;
+    @include animation(time, 60s);
+  }
+
+  &__second {
+    width: 10px;
+    height: 249px;
```

Comment:
For now it's some magic numbers
Will be better to calculate the size for all hands depending on some watch size

Reply:
Okey. I got it